datareduction/datareducer.cpp

```diff
@@ -539,7 +539,35 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
          continue;
       }
       if(*it == "fg_HallE") {
```

Comment:
HallE or fg_HallE?

Reply:
Ah, yes, these should ideally match with the config names wherever possible. Changed.

Reply:
Where?

Reply:
Here.

Reply:
Err, silly github. See the commit below.

Reply:
I watch the diff and line 541 hasn't changed.

Reply:
I think Yann meant something like
if(*it == "HallE" || *it == "fg_HallE") {
?

Reply:
Ah. D'oh. Indeed. Just a second.